test/test_force.js

```diff
@@ -157,7 +157,8 @@ function main(args) {
 
             if (testingWithTemplate) {
                 // NB: chosenAppTypes[0] is appType from template
-                createCompileApp(tmpDir, os, chosenAppTypes[0], templateRepoUri, pluginRepoUri, useSfdxRequested);
+                var appType = chosenAppTypes.length > 0 ? chosenAppTypes[0] : [templateHelper.getAppTypeFromTemplate(templateRepoUri)];
```

Comment:
`chosenAppTypes[0]` does not get set (line 90) if sfdx is used, which causes a null pointer exception in createCompileApp.